worker/src/r2/builder.rs

```diff
@@ -164,12 +164,12 @@ impl<'bucket> PutOptionsBuilder<'bucket> {
     }
 
     /// A map of custom, user-defined metadata that will be stored with the object.
-    pub fn custom_metdata(mut self, metadata: impl Into<HashMap<String, String>>) -> Self {
+    pub fn custom_metadata(mut self, metadata: impl Into<HashMap<String, String>>) -> Self {
```

Comment:
This change (and the same on Line 233 of the same file) would be a breaking change as the function name exposed to the user has changed.

Reply:
That's unfortunate..... I'm fine with the break change since 0.1.0 is going to have quite a few of them.